test/celeritas/optical/Cerenkov.test.cc

```diff
@@ -35,10 +35,12 @@
 
 namespace celeritas
 {
+namespace optical
+{
 namespace test
 {
-using namespace celeritas::optical;
-
+//---------------------------------------------------------------------------//
+//
```

Comment:
Nitpick: Missing comment or extraneous `//`?

Reply:
I'll pull that in on the next PR (which actually tackles cerenkov issue)